theme/migrations/0033_auto_20240201_1947.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.20 on 2024-02-01 19:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('theme', '0032_alter_userquota_grace_period_ends'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='quotamessage',
+            name='enforce_content_prepend',
+            field=models.TextField(default='You can not take further action because you have exceeded your quota. Your quota for HydroShare resources is {allocated}{unit}. You currently have resources that consume {used}{unit}, {percent}% of your quota. '),
```

Comment:
I think the first part of both of these paragraphs is now redundant and can be removed, given that we now have a detailed representation of the allocation in the profile page.

Reply:
Agreed that it feels redundant when one is on their profile page. However, these messages are also used for messaging in other places:
* immediately after login, on the home page (https://github.com/hydroshare/hydroshare/blob/5228-hard-quota-enforcement/theme/views.py#L785)
* in this task, which sends the quota_message in an email (https://github.com/hydroshare/hydroshare/blob/5228-hard-quota-enforcement/hs_core/tasks.py#L570)


In these cases, the redundant part of the message feels more important.
Worth mentioning these messages are stored in the db and edited in the admin panel, so not limited to what we have here in this migration.

Reply:
I see, in that case perhaps we could make an exception for the profile page and have it render its own message in HTML.